### DISCUSSION TITLE
supabase.auth.getSession warning when not used in Code

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#31155] [Questions] [answered]
I get this warning all the time : 

Using supabase.auth.getSession() is potentially insecure as it loads data directly from the storage medium (typically cookies) which may not be authentic. Prefer using supabase.auth.getUser() instead. To suppress this warning call supabase.auth.getUser() before you call supabase.auth.getSession().

I am not using supabase.auth.getSession at all anywhere in the code, why do I get this warning message

**Comment by GaryAustin1** [comment, marked as answer]
It is used somewhere in the Supabase code and the warning is not discerning enough to know that it is an OK case.
https://github.com/supabase/auth-js/issues/888#issuecomment-2093787893
https://github.com/supabase/auth-js/issues/912
Other threads here in discussions on it too.
